go.mod

```diff
@@ -1,5 +1,8 @@
 module github.com/tilezen/go-tilepacks
 
-go 1.12
+require (
+	github.com/aaronland/go-string v0.1.0
```

Comment:
What does this new requirement provide?

Reply:
`StringToDSNWithKeys` &etc?

Reply:
It's generic code for parsing `foo=bar hello=world` DSN strings in to maps.